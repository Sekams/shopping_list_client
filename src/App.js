import React, { Component } from 'react';
import NavBar from './components/nav_bar'
// import LoginFrom from './components/login_form'
// import RegistrationForm from './components/registration_form'
// import ShoppingListItem from './components/shopping_list_item'
// import ShoppingList from './components/shopping_list'
// import HomePage from './components/home_page'
import SnackBar from './components/snackbar'

class App extends Component {
  constructor(props) {
    super(props);

    this.state = {
      username: 'Sekams',
      authorized: true,
      logged_in: true,
      page: 'home'
    };
  }

  render() {
    var toiletries = [
      {
        name: "Soap",
        price: "200",
        status: true
      },
      {
        name: "Deodorant",
        price: "300",
        status: false
      },
      {
        name: "Shaver",
        price: "100",
        status: false
      },
      {
        name: "Perfume",
        price: "600",
        status: true
      }
    ]
    var groceries = [
      {
        name: "Tomatoes",
        price: "20",
        status: false
      },
      {
        name: "Onions",
        price: "50",
        status: true
      }
    ]
    var shoppingLists = [
      {
        title: "Toiletries",
        shoppingListItems: toiletries
      },
      {
        title: "Groceries",
        shoppingListItems: groceries
      }
    ]
    return (
      <div>
        <NavBar
          username={this.state.username}
          authorized={this.state.authorized}
          logged_in={this.state.logged_in}
          page={this.state.page} />
        {/* <LoginFrom /> */}
        {/* <RegistrationForm /> */}
        {/* <HomePage
          shoppingLists={shoppingLists} /> */}
        <SnackBar
          class="success-snackbar"
          message="It works" />
      </div>
    );
  }
}

export default App;
