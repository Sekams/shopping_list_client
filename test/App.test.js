import React from 'react';
import App from '../src/App';

test('renders the app component', () => {
  const wrapper = shallow(
    <App />
  );
  expect(wrapper).toMatchSnapshot();
});
