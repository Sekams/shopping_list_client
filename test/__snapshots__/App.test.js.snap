// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the app component 1`] = `
<BrowserRouter>
  <Main />
</BrowserRouter>
`;
